inc/customizer.php

```diff
@@ -94,7 +94,19 @@ function primer_customize_register( $wp_customize ) {
 		'priority' => 1,
 	) );
 
-	// Add page background color setting and control.
+	// Add menu background color setting and control.
+	$wp_customize->add_setting( 'menu_background_color', array(
+		'default'           => $color_scheme[3],
+		'sanitize_callback' => 'sanitize_hex_color',
+		'transport'         => 'postMessage',
+	) );
+
+	$wp_customize->add_control( new WP_Customize_Color_Control( $wp_customize, 'menu_background_color', array(
+		'label'       => __( 'Menu Background Color', 'primer' ),
+		'section'     => 'colors',
+	) ) );
+
+	// Add tagline text color setting and control.
 	$wp_customize->add_setting( 'tagline_text_color', array(
 		'default'           => $color_scheme[2],
```

Comment:
@chriswallace This is actually a cascading problem, all of the `$wp_customize->add_setting` methods will need to have their defaults updated to use the correct index as defined in `primer_get_color_schemes()`